client/src/templates/Challenges/codeally/show.tsx

```diff
@@ -169,14 +169,18 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
     const {
       data: {
         challengeNode: {
-          challenge: { url, coderoadTutorial }
+          challenge: { url }
         }
       }
     } = this.props;
 
+    const repoUrl = `https://github.com/${url}`;
+    const coderoadTutorial = encodeURIComponent(
+      `https://raw.githubusercontent.com/${url}/main/tutorial.json`
+    );
```

Comment:
I changed the URL in the challenge files to just `freeCodeCamp/learn-world-cup-database`. So `repoUrl` should be `https://github.com/freeCodeCamp/learn-number-guessing-game` and `coderoadTutorial` should be ``encodeURIComponent(`https://raw.githubusercontent.com/freeCodeCamp/learn-world-cup-database/main/tutorial.json`);``

It would probably be better to change the name in the challenge meta to `repoName` or something.

Reply:
So you did. I just didn't see it in the files diff earlier.

Reply:
> It would probably be better to change the name in the challenge meta to repoName or something.

Yeah, that works. Separate PR's fine.